firstvoices/backend/serializers/base_serializers.py

```diff
@@ -203,3 +203,10 @@ class WritableControlledSiteContentSerializer(
 
     class Meta(BaseControlledSiteContentSerializer.Meta):
         fields = BaseControlledSiteContentSerializer.Meta.fields
+
+
+class LinkedSiteMinimalSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Site
+        fields = ("id", "title", "slug")
+        read_only_fields = ("id", "title", "slug")
```

Comment:
The site info is standard across all the serializers, so I'd be inclined to keep it the way it was (I think just language__title and visibility added here).